cluster/images/hyperkube/Makefile

```diff
@@ -15,44 +15,14 @@
 # Build the hyperkube image.
 #
 # Usage:
-#   [ARCH=amd64] [REGISTRY="gcr.io/google_containers"] make (build|push) VERSION={some_released_version_of_kubernetes}
+#   [ARCH=amd64] [REGISTRY="gcr.io/google-containers"] make (build|push) VERSION={some_released_version_of_kubernetes}
 
-REGISTRY?=gcr.io/google_containers
+REGISTRY?=gcr.io/google-containers
```

Comment:
that ship has already sailed...
```console
$ sift --no-group "gcr\\.io/google-containers" | wc -l
95
$ sift --no-group "gcr\\.io/google_containers" | wc -l
526
```

I've considering doing a `s/google_containers/google-containers/`, though I'm not sure what I might break. :P